### PR TITLE
Modify admission to validate SubnamespaceAnchor

### PR DESCRIPTION
**Pull request by akazs**

### CODE REVIEW

**Review by zoetrope — APPROVED** [review]
LGTM

File: admission/hooks/validate_subnamespaceanchor_test.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+		Version: "v1alpha2",
+		Kind:    "SubnamespaceAnchor",
+	})
+	subns.SetName(name)
+	subns.SetNamespace("default")
+	subns.UnstructuredContent()["spec"] = map[string]interface{}{}
+	return subns
+}
+
+var _ = Describe("validate SubnamespaceAnchor webhook with ", func() {
```
Comment (masa213f) [inline admission/hooks/validate_subnamespaceanchor_test.go:24]: Is there anything word after `with `?

Comment (akazs) [reply admission/hooks/validate_subnamespaceanchor_test.go:24]: No, but there are several other tests using similar sentences.

Comment (masa213f) [reply admission/hooks/validate_subnamespaceanchor_test.go:24]: I see.

**Pull request merged**